### DISCUSSION TITLE
VirusTotal and Hybrid-Analysis scan results

### DISCUSSION THREAD

**Discussion by ghost** [legendary-gl/legendary#207] [General]
Hi, I'm quite new to this program and I just found this on Reddit.

I decided to run **VirusTotal** and **Hybrid-Analysis** on the .exe file for Windows and these are the results;

- https://www.virustotal.com/gui/file/1d8d148a39f8a1af30ca270251aa7a999ba30c474bbfc79274399cf5b293403f/detection
- https://www.hybrid-analysis.com/sample/1d8d148a39f8a1af30ca270251aa7a999ba30c474bbfc79274399cf5b293403f

**What are these detected red flags?**

Well. I know the program is open-source. I just want to re-assure though.

**Comment by derrod** [comment]
False positives by shit antivirus software.

**Comment by TabulateJarl8** [comment]
It seems that most things created via PyInstaller are detected as malicious by some antivirus for some reason. I've noticed this with some of my past projects and like derrod said, it's just bad quality antivirus detecting it as a false positive.
